packages/core/src/components/cv-data-table/cv-data-table-row.vue

```diff
@@ -51,11 +51,11 @@ export default {
   },
   mounted() {
     // NOTE: this.$slots is not reactive so needs to be managed on beforeUpdate
-    this.dataExpandable = this.$slots.expandedContent !== undefined;
+    this.dataExpandable = !!(this.$slots.expandedContent !== undefined);
```

Comment:
I guess here it is already Boolean, so no need for `!!` or?

Reply:
Good point, that is a danger of find and replace. Will correct.